src/kernels/kernel.node.ts

```diff
@@ -67,6 +67,13 @@ export class Kernel extends BaseKernel {
     }
 
     protected async getDebugCellHook(): Promise<string[]> {
+        const useNewDebugger = this.configService.getSettings(undefined).useJupyterDebugger === true;
+        if (useNewDebugger) {
```

Comment:
Added a hidden setting that provides a way for us (jupyter engineers) to test the jupyter debugger even with local kernels.
Just like we test zmq.